packages/workflow/src/interceptors.ts

```diff
@@ -195,8 +195,6 @@ export interface DisposeInput {}
  * Interceptor for the internals of the Workflow runtime.
  *
  * Use to manipulate or trace Workflow activations.
- *
- * @experimental
```

Comment:
Let's keep this one experimental, this is really for advanced use cases and we should reserve the right to modify our internal at will.

Reply:
Fair enough